spacy_llm/tests/backends/test_anthropic.py

```diff
@@ -0,0 +1,23 @@
+# mypy: ignore-errors
+import pytest
+from typing import Iterable
+
+from spacy.tokens import Doc
+
+from ...registry import registry
+from ..compat import has_anthropic_key
```

Comment:
Only use the relative imports in the test suite if it's other modules from the test suite, which is the case for `compat` here. For `registry`, we should import explicitely from `spacy_llm.registry`.